backend/app/api/health.py

```
@@ -1,17 +1,22 @@
 router = APIRouter(tags=["health"])
 
 
 @router.get("/health")
-async def health_check():
-    """서버 상태 및 인메모리 데이터 현황."""
+async def health_check(db: AsyncSession = Depends(get_db)) -> dict:
+    """서버 상태."""
+    try:
+        await db.execute(text("SELECT 1"))
+        db_ok = True
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)